spec/models/transaction_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../rails_helper'
+
+RSpec.describe Transaction, type: :model do
```

Comment:
- [ ] As requested in the project the app should have unit tests and integration tests for all the most important components of the RoR application 🙃. 
- [ ] When I run the test I get an error, instead of `Transaction` you should write `TracePoint`, and also please make sure all your tests are passing 😉 

<img width="762" alt="Screen Shot 2023-03-01 at 17 25 09" src="https://user-images.githubusercontent.com/74501523/222171949-d948060f-0265-43a9-a943-3cb6f4b0bbc2.png">